iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift

```diff
@@ -148,34 +124,102 @@ class HomeViewController: UIViewController {
   private func handleTokenResponse(with result: Result<TokenDetails, TokenRequestError>) {
     switch result {
       case .failure(let error):
-        showAlert(with: error.localizedDescription)
+        showAlert(with: error.localizedDescription, title: "Error")
       case .success(let tokenDetails):
-        showAlert(with: tokenDetails.token)
+        showAlert(with: tokenDetails.token, title: "Success")
     }
   }
 
-  private func showAlert(with cardToken: String) {
+  private func showAlert(with message: String, title: String = "Payment") {
     DispatchQueue.main.async {
       let alert = UIAlertController(title: "Payment",
-                                    message: cardToken, preferredStyle: .alert)
+                                    message: message,
+                                    preferredStyle: .alert)
       let action = UIAlertAction(title: "OK", style: .default) { _ in
-        alert.dismiss(animated: true, completion: nil)
+        alert.dismiss(animated: true)
       }
       alert.addAction(action)
-      self.present(alert, animated: true, completion: nil)
+      self.present(alert, animated: true)
+    }
+  }
+
+  // MARK: - Apple Pay
+
+  private func handleApplePayAction() {
```

Comment:
I wonder if we should have this method at all. WIth this being demo code, we are having:
- button press
- function to decide what to do
- processing outcome

The button press is a single line, calling for this function, so we could just handle this there?
- button press 
- processing outcome